files/en-us/web/accessibility/index.html

```diff
@@ -35,6 +35,8 @@ <h2 id="Other_documentation">Other documentation</h2>
 <dl>
  <dt><a href="/en-US/docs/Web/Accessibility/Understanding_WCAG">Understanding the Web Content Accessibility Guidelines</a></dt>
  <dd>This set of articles provides quick explanations to help you understand the steps that need to be taken to conform to the recommendations outlined in the W3C Web Content Accessibility Guidelines 2.0 (WCAG 2.0 or just WCAG, for the purposes of this writing).</dd>
+ <dt><a href="/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance">Introduction to Color and Accessibility</a></dt>
```

Comment:
We are using sentence case convention on MDN. Besides this, I'm extremely happy you catched this!
```suggestion
 <dt><a href="/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance">Introduction to color and accessibility</a></dt>
```

Reply:
Ooops on the sentence case, I'll keep that in mind... some habits die hard LOL!